packages/shopify-app-remix/src/server/clients/admin/graphql.ts

```diff
@@ -55,12 +46,11 @@ export function graphqlClientFactory({
         headers: flatHeaders(apiResponse.headers),
       });
     } catch (error) {
-      throw await handleClientError({
-        params: {api, config, logger},
-        request,
-        error,
-        shop: session.shop,
-      });
+      if (handleClientError) {
+        throw await handleClientError({error, params, session});
+      }
+
+      throw new Error(error);
```

Comment:
I think it might mess up the stack trace if we throw a new error at this point - would it be possible to re-throw `error`?

Reply:
Fixed in `reset-unauth-admin-client`